package.json

```diff
@@ -143,6 +142,7 @@
     "worker-loader": "^2.0.0"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-optional-chaining": "^7.10.4",
```

Comment:
`preset-env` already includes this on by default as of 7.8.0, and we've recently explicitly bumped it to 7.10.x

```suggestion
```

Reply:
Oops I wasn't aware of this 😬 I've removed the plugin now